packages/react-dom/src/__tests__/ReactMountDestruction-test.js

```diff
@@ -46,6 +57,7 @@ describe('ReactMount', () => {
         <div />
       </div>
     );
+    // Cannot be migrated to createRoot until we remove unmountComponentAtNode i.e. remove this test.
```

Comment:
Do we have a test for if you try to createRoot on a container that React has rendered to? Might be worth checking. 

Also curious if we have a test that removes the container passed to `createRoot` and then we call unmount on the non-existent container. 

I know these are kinda unrelated, but as I'm going through these tests we're changing I'm also looking for possible missing tests in similar things for `createRoot`.

Reply:
Good point. I'll check if we have tests though it wouldn't be a very accurate search. But it does make sense to me to have tests for these cases.

Reply:
> Do we have a test for if you try to createRoot on a container that React has rendered to? Might be worth checking.

We have [`ReactDOMRoot - warns when creating two roots managing the same container`](https://github.com/facebook/react/blob/a6dce069a38baadb20515b571a48f750021a72b4/packages/react-dom/src/__tests__/ReactDOMRoot-test.js#L342). Does that match what you are looking for? We also have a test in the same file if we render into existing markup.

> Also curious if we have a test that removes the container passed to createRoot and then we call unmount on the non-existent container.

I added a test if we unmount after something else cleared the container content. We already had a test for updating but not unmounting. Does that match what you were looking for?

Reply:
Yup both sound good to me.